packages/framework-base/src/component-patterns/README.md

```diff
@@ -0,0 +1,39 @@
+# `fluentui-react-native` - Common component patterns
+
+These are the base component patterns shared across the deprecated framework, and the newer framework. This also includes the custom JSX handlers required to render them properly.
+
+There are two main patterns exposed here: direct rendering and staged rendering.
+
+## Direct Rendering
+
+The direct rendering pattern allows a component to be called directly, rather than creating a new entry in the DOM.
+
+As an example, if you want to create a wrapper around a component called `MyText` that has `italicize` as one of its props, that always wants to set that value to true. You could define:
+
+```ts
+const MyNewText: React.FunctionComponent<MyTextProps> = (props) => {
+  return <MyText {...props, italicize: true} />;
+}
+```
+
+When this is rendered, there is an entry for `MyNewText` which contains a `MyText` (another entry), which might contains `Text` (for react-native usage). The direct rendering pattern is one where a component can denote that it is safe to be called directly as a function, instead operating as a prop transform that gets applied to the underlying component.
+
+- For the above to be safe, `MyNewText` should NOT use hooks. In the case of any conditional rendering logic this will break the rule of hooks.
```

Comment:
It can totally safely use hooks though? At least, that one small example provided above?

Reply:
Yes, the pattern is designed with that in mind. I will make the documentation more comprehensive in the next stage of the change and remove the older documentation which is currently still existing.

Reply:
yeah some clarity here would be nice. This makes it sound like hooks can't be used with FURN components when used as direct components at all, which sounds bad?